tests/system/feature_store/test_spark_engine.py

```diff
@@ -80,3 +86,108 @@ def test_basic_remote_spark_ingest(self):
             spark_context=self.spark_service,
             run_config=fs.RunConfig(local=False),
         )
+
+    @pytest.mark.parametrize("partitioned", [True, False])
+    def test_schedule_on_filtered_by_time(self, partitioned):
+        name = f"sched-time-{str(partitioned)}"
+
+        now = datetime.now() + timedelta(minutes=2)
+
+        path = "v3io:///bigdata/bla.parquet"
+        fsys = fsspec.filesystem(v3iofs.fs.V3ioFS.protocol)
```

Comment:
Any reason not to use store_manager.get_or_create_store & get_filesystem?

Reply:
not really